modin/pandas/test/dataframe/test_reduction.py

```diff
@@ -312,6 +312,38 @@ def test_prod_specific(min_count, numeric_only):
     )
 
 
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+@pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
+@pytest.mark.parametrize(
+    "skipna", bool_arg_values, ids=arg_keys("skipna", bool_arg_keys)
+)
+@pytest.mark.parametrize(
+    "numeric_only", bool_arg_values, ids=arg_keys("numeric_only", bool_arg_keys)
+)
+@pytest.mark.parametrize("ddof", int_arg_values, ids=arg_keys("ddof", int_arg_keys))
+def test_sem(data, axis, skipna, numeric_only, ddof):
```

Comment:
Better place for the test is `test_window.py`.

You also can add `sem` test for #1953 issue to `test_median_skew_std_var_1953` test.

Reply:
Moved into `test_window.py`, added the test to `test_median_skew_std_var_1953`. Thanks!